ui/nuxeo-document-comments/nuxeo-document-comment.js

```diff
@@ -52,7 +52,7 @@ import { FormatBehavior } from '../nuxeo-format-behavior.js';
   class DocumentComment extends mixinBehaviors([FormatBehavior], Nuxeo.Element) {
     static get template() {
       return html`
-        <style include="nuxeo-document-comments-styles">
+        <style include="nuxeo-document-comments-styles nuxeo-styles">
```

Comment:
Not sure if we want this approach since `nuxeo-styles` is something defined on Web UI and when we have the elements by themselves, we won't have those styles.
If I am seeing it in the correct way, we can "expose" vars/mixins on our elements in order to customize them, but probably not having this kind of relation/dependency. 

@nelsonsilva , would you be able to provide us further details?

Reply:
yeah I need some help understanding this

Reply:
I removed the include nuxeo-styles